packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuItemsContainer.tsx

```diff
@@ -24,7 +23,7 @@ const StyledScrollWrapper = styled(ScrollWrapper)`
 `;
 
 const StyledDropdownMenuItemsInternalContainer = styled.div`
-  align-items: flex-start;
+  align-items: stretch;
```

Comment:
we will need to double check chromatic tests here to make sure this change does not create a visual regression